services/QuillLMS/app/models/user.rb

```diff
@@ -860,5 +861,9 @@ def school_or_district_premium?
   private def update_invitee_email_address
     Invitation.where(invitee_email: email_before_last_save).update_all(invitee_email: email)
   end
+
+  private def generate_default_notification_email_frequency
+    TeacherInfo.create!(user: self, notification_email_frequency: TeacherInfo::DAILY_EMAIL)
```

Comment:
I would use:
`teacher_info.create!(notification_email_frequency: TeacherInfo::DAILY_EMAIL)`

since it highlights the has_one relationship with teacher_info and has the implicit user context.